### PR TITLE
Create header & main section 

### PR DESCRIPTION
**Pull request by PabloEGonz**
I created  the header with log and menu  bar, main-section with background image.

### CODE REVIEW

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @joramot , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark:  
- [x] Good commit messages :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:    


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <a
+              ><img
+                class="icons"
+                src="/images/medium-icon.png"
+                alt="medium-icon"
+            /></a>
+          </li>
+        </ul>
+      </section>
+    </main>
```
[the remark this answers is not in the document]
Comment (PabloEGonz) [reply index.html]: @Felix45 Hi there! I tried to use the correct font family but stylelint was marking them as errors, what should I do in this case?

**Comment by joramot** [comment]
Hi Paul, possibly you have to declare a font family in addition to the one
you want to use, it is good practice to specify two types of font families,
example:

font-family: roboto, san-serif;

That is the most common error that the linters tool tells you.

I hope my advice will help you.Greetings.

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @PabloEGonz, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**